modin/pandas/groupby.py

```diff
@@ -82,7 +82,7 @@
 
 
 @_inherit_docstrings(pandas.core.groupby.DataFrameGroupBy)
-class DataFrameGroupBy(ClassLogger):
+class DataFrameGroupBy(ClassLogger, modin_layer="GROUPBY"):
```

Comment:
```suggestion
class DataFrameGroupBy(ClassLogger):
```
This is also pandas API.